internal/cmd/schema-registry/command_schema_describe.go

```diff
@@ -223,7 +224,21 @@ func printSchema(cmd *cobra.Command, schemaID int64, schema string, sType string
 	if sType != "" {
 		utils.Println(cmd, "Type: "+sType)
 	}
-	utils.Println(cmd, "Schema: "+schema)
+
+	switch sType {
+	case "PROTOBUF":
+		utils.Println(cmd, "Schema:")
+		utils.Println(cmd, schema)
+	default:
+		var jsonBuffer bytes.Buffer
+		if err := json.Indent(&jsonBuffer, []byte(schema), "", "    "); err != nil {
+			return err
+		}
+		schemaString := jsonBuffer.String()
+		utils.Println(cmd, "Schema:")
+		utils.Println(schemaString)
+	}
```

Comment:
I'm assuming that "JSON" and "AVRO" are the only other types here. If there's ever a new type in the future we should not treat it as JSON by default. We can also get rid of some duplication here too.
```suggestion
	switch sType {
	case "JSON", "AVRO":
		var jsonBuffer bytes.Buffer
		if err := json.Indent(&jsonBuffer, []byte(schema), "", "    "); err != nil {
			return err
		}
		schema = jsonBuffer.String()
	}
	
	utils.Println(cmd, "Schema:")
	utils.Println(schemaString)
```